### DISCUSSION TITLE
Gaines not related to trades

### DISCUSSION THREAD

**Discussion by roadbridgeaward** [rotki/rotki#5629] [Archive]
Does rotki recognize the amounts of assets that the user gained on Binance but not via spot trades?
For example, does rotki retrieve from Binance the assets that the user gained due to subscribing in flexible savings / earning? Does rotki include such gains  in the PnL report?

**Comment by LefterisJP** [comment]
hey I believe you asked this in discord already and the issue you are talking about is this: https://github.com/rotki/rotki/issues/1505

**Reply by roadbridgeaward** [reply]
Thank you so much
